src/Humanizer/DateToOrdinalWordsExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Globalization;
+using Humanizer.Configuration;
+
+namespace Humanizer
+{
+    /// <summary>
+    /// Humanizes DateTime into human readable sentence
+    /// </summary>
+    public static class DateToOrdinalWordsExtensions
+    {
+        /// <summary>
+        /// Turns the provided date into ordinal words
+        /// </summary>
+        /// <param name="input">The date to be made into ordinal words</param>
+        /// <returns>The date in ordinal words</returns>
+        public static string ToOrdinalWords(this DateTime input)
+        {
+            return input.Day.Ordinalize() + input.ToString(" MMMM yyyy");
```

Comment:
Yes, just provide as minimum the implementation for English, other languages can then localize this functionality with own PRs. See for example #493